libjulia/backends/evm/EVMCodeTransform.h

```diff
@@ -87,11 +100,14 @@ class CodeTransform: public boost::static_visitor<>
 	void operator()(solidity::assembly::StackAssignment const& _assignment);
 	void operator()(solidity::assembly::Assignment const& _assignment);
 	void operator()(solidity::assembly::VariableDeclaration const& _varDecl);
-	void operator()(solidity::assembly::Block const& _block);
 	void operator()(solidity::assembly::Switch const& _switch);
 	void operator()(solidity::assembly::FunctionDefinition const&);
+	void operator()(solidity::assembly::Block const& _block);
+
+	AbstractAssembly::LabelID labelFromIdentifier(solidity::assembly::Identifier const& _identifier);
+	/// Generates code for an expression that is supposed to return a single value.
+	void visitExpression(solidity::assembly::Statement const& _expression);
 
-private:
```

Comment:
I would like the visitor functions to be private, but that seems to be quite a hassle...
Made the others private at least.